feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt

```diff
@@ -132,6 +201,27 @@ fun PreviewTimetableGridItem() {
             TimetableGridItem(
                 timetableItem = Session.fake(),
                 onTimetableItemClick = {},
+                gridItemHeightPx = 350,
+            )
+        }
+    }
+}
+
+@Preview
+@Composable
+fun PreviewTimetableGridLongTitleItem() {
```

Comment:
Can I see how the text becomes small? In this preview, we see normal size text right?

Reply:
@shxun6934 Thanks for your quick action! 
It might be a reply after tomorrow(today) morning 😉